style.css

```diff
@@ -0,0 +1,100 @@
+body {
+  background: grey;
+  background: url("https://images.unsplash.com/photo-1484100356142-db6ab6244067?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=400&q=60");
+}
+
+.survey {
+  display: block;
+  margin: 0 auto;
+}
+
+.survey-form {
+  display: block;
+  margin: 0 auto;
+
+  /* border:10px solid greenyellow; */
```

Comment:
Remove the commented code

Reply:
- Here we can say politely something like:
  * It would be nice to remove comments to have a clean code.